saturday-afternoon/components/Legs/Leg.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+
+import * as React from "react";
+import { Col, Row, Card } from "antd";
+
+import resolveScopedStyles from "../../utils/resolveScopedStyles";
+
+const cardStyles = resolveScopedStyles(
+  <scope>
+    <style jsx>{`
+      .card {
+        margin-top: 6px;
+      }
+    `}</style>
+  </scope>
+);
+
+const Leg = () => (
+  <Card
+    title="Mon 26 Mar — Tue 27 Mar"
+    className={`card ${cardStyles.className}`}
```

Comment:
className is confusing here. it is not seen from the first sight where it comes from